cookbook/security/pre_authenticated.rst

```diff
@@ -11,6 +11,14 @@ box, Symfony supports most authentication mechanisms.
 These requests are called *pre authenticated* requests because the user is already
 authenticated when reaching your application.
 
+.. caution::
+
+    :doc:`User impersonation </cookbook/security/impersonating_user>` is not
+    compatible with pre Authenticated firewalls. The reason is that
```

Comment:
It's what I would write also, but the title of the cookbook is "Using pre Authenticated Security Firewalls" so I wanted to be consistent.
https://symfony.com/doc/2.7/cookbook/security/pre_authenticated.html

Should it be changed everywhere?